apps/nginx/.config

```diff
@@ -1,12 +1,12 @@
 #
 # Automatically generated file; DO NOT EDIT.
-# Unikraft/0.9.0~539cbf1-custom Configuration
+# Unikraft/0.9.0~8ed9279-custom Configuration
 #
-CONFIG_UK_FULLVERSION="0.9.0~539cbf1-custom"
+CONFIG_UK_FULLVERSION="0.9.0~8ed9279-custom"
 CONFIG_UK_CODENAME="Hyperion"
 CONFIG_UK_ARCH="x86_64"
-CONFIG_UK_BASE="/home/masa/work/unikraft/development/unikraft"
-CONFIG_UK_APP="/home/masa/work/unikraft/development/apps/nginx"
+CONFIG_UK_BASE="/scratch/okelmann/unikraft-development/unikraft"
+CONFIG_UK_APP="/scratch/okelmann/unikraft-development/apps/nginx"
```

Comment:
I wonder if there is a good way to handle these abs paths... I will merge this PR anyway

Reply:
the .configs from eurosys21-evaluation also contain absolute paths there. I think ignoring it is easier that fixing it.

Reply:
yeah, let's keep it this way for now.
(maybe we can create a mount namespace so that this repository locates at the same location?)